Dockerfile.15sp0

```diff
@@ -0,0 +1,31 @@
+FROM registry.scc.suse.de/suse/sles15:ga-beta7
```

Comment:
Maybe we should set the tag `sles15:ga` here, and in the registry tag the most recent image with this tag.

Reply:
Yeah, I wasn't too sure about that.
On one hand, I like the explicitness of `sles15:ga-beta7` -- that's the version we are currently testing. The downside is that we'll have to update the dockerfile as soon as RC1, RC2,..., GA are released.
On the other hand, using `sles15:ga` we can leave the dockerfile unchanged forever, but we don't *really* know if the image is a beta/rc or actually GA (until GA gets released).
`docker-ls` shows:
```
> docker-ls tags suse/sles15 --registry http://registry.scc.suse.de
requesting list . done
repository: suse/sles15
tags:
- ga
- ga-beta6
- ga-beta7
- sp0
```

TBH I kind of wish we didn't have those `sp0` and `ga` tags UNTIL those products became official... Right now, I have no idea on which beta `ga` and `sp0` got tagged, so I can't rely on them.

Reply:
Well, it's not too hard to tell which image it is:

```
$ docker-ls tag suse/sles15:ga --registry http://registry.scc.suse.de
requesting manifest . done
repository: suse/sles15
tagName: ga
digest: sha256:a04db4ae11a6499c9ad407adbf7d84030f45640b789842628b9e0b0b98e9be51
layers:
- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
- sha256:e285d47f6e1f15e96218a62151440a8519d39e433d5ea2f20611cd1a7f17b3bf
```

```
$ docker-ls tag suse/sles15:ga-beta7 --registry http://registry.scc.suse.de
requesting manifest . done
repository: suse/sles15
tagName: ga-beta7
digest: sha256:4064d0802957330de7f800d80cd484025d5a3b373c7a7a8425a82f57d1b3edcf
layers:
- sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4
- sha256:e285d47f6e1f15e96218a62151440a8519d39e433d5ea2f20611cd1a7f17b3bf
```

That's the same image, which makes sense. :grimacing:

Reply:
Could have been, could not have been. Depends on who tagged them and when. It's not so cool having to look at layer digests to check if they're the same image.. that's the whole point of the tag I guess.
Basically my problem is that the tag is `ga` but the image is not really GA, it's beta 7.

Reply:
Updating the tag in the file is also fine, just something we need not to forget to do.

Reply:
Yup :)